manifests/extract_file.pp

```diff
@@ -0,0 +1,12 @@
+define solr5::extract_file(
+  $path_to_archive,
+  $file_to_extract,
+  $archive_filename=$name,
+){
+  exec { $name:
+    command => "tar --extract --file ${path_to_archive}/${archive_filename} --wildcards --no-anchored '${file_to_extract}' -O > ${path_to_archive}/${file_to_extract} ",
+    cwd     => $path_to_archive,
+    path    => '/bin/',
```

Comment:
Because the line is long enough even without the path, besides I think it adds just noise.